docs/src/components/App.js

```diff
@@ -10,18 +10,41 @@ type Props = {|
   children?: React.Node,
 |};
 
+const localStorageOrganizedByKey = 'gestalt-sitebar-organized-by';
```

Comment:
nit: is this a typo?  `gestalt-**sidebar**-organized-by`

Reply:
@AlbertCarreras thanks for spotting - updated all of these